exercises/practice/roman-numerals/roman_numerals_test.go

```diff
@@ -6,7 +6,7 @@ func TestRomanNumerals(t *testing.T) {
 	tc := []romanNumeralTest{
 		{0, "", true},
 		{-1, "", true},
-		{3001, "", true},
+		{4000, "", true},
```

Comment:
I agree with adding this test, as it won't break any existing good solutions and does properly check if the solution is doing some kind of checks to the number.

Numbers equal or bigger than `4000` will be an error, whether someone is just checking for values lower `3000` influenced by the description or for values lower than `4000` if they read a bit more on the topic.